app/assets/javascripts/visualizations/highvis/histogram.coffee

```diff
@@ -250,120 +241,89 @@ $ ->
 
             options =
               showInLegend: false
-              color: globals.configs.colors[groupIndex % globals.configs.colors.length]
+              color: globals.getColor(groupIndex)
               name: data.groups[groupIndex]
               data: finalData
 
             @chart.addSeries options, false
 
-        @chart.xAxis[0].setExtremes @globalmin - (@configs.binSize / 2), @globalmax + (@configs.binSize / 2), false
+        @chart.xAxis[0].setExtremes(@globalmin - (@configs.binSize / 2),
+          @globalmax + (@configs.binSize / 2), false)
 
         @chart.redraw()
 
       buildLegendSeries: ->
         count = -1
-        for field, fieldIndex in data.fields when fieldIndex in data.normalFields
+        for f, fi in data.fields when fi in data.normalFields
```

Comment:
This is inconsistent: in most other places, you'd use something like `f, i` instead of `f, fi`